packages/react-instantsearch/src/core/createInstantSearchServer.js

```diff
@@ -53,17 +53,20 @@ const createInstantSearch = function(algoliasearch) {
 
     const mergedSearchParameters = searchParameters
       .filter(searchParameter => hasMultipleIndex(searchParameter.context))
-      .reduce((acc, searchParameter) => {
-        const index = getIndex(searchParameter.context);
-        const sp = searchParameter.getSearchParameters.call(
-          { context: searchParameter.context },
-          acc[index] ? acc[index] : sharedSearchParameters,
-          searchParameter.props,
-          searchParameter.searchState
-        );
-        acc[index] = sp;
-        return acc;
-      }, {});
+      .reduce(
+        (acc, searchParameter) => {
```

Comment:
That's probably because the hook is using the correct version of prettier, and your editor the system one, which might be outdated?